docs/providers/Analytics.tsx

```diff
@@ -35,6 +43,17 @@ export function AnalyticsProvider(props: AnalyticsProps) {
         gtag('js', new Date());
         gtag('config', '${MEASUREMENT_ID}', { 'transport_type': 'beacon', 'anonymize_ip': true });
       `}</Script>
+      <Script
+        id="gtm-script"
+        strategy="lazyOnload"
+        src={`https://www.googletagmanager.com/gtag/js?id=${GA4_MEASUREMENT_ID}`}
+      />
+      <Script id="gtm-init" strategy="lazyOnload">{`
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', '${GA4_MEASUREMENT_ID}', { 'transport_type': 'beacon', 'anonymize_ip': true });
+      `}</Script>
```

Comment:
```suggestion
      <Script id="ga4-init" strategy="lazyOnload">{`
        window.dataLayer = window.dataLayer || [];
        function ga4tag(){dataLayer.push(arguments);}
        ga4tag('js', new Date());
        ga4tag('config', '${GA4_MEASUREMENT_ID}', { 'transport_type': 'beacon', 'anonymize_ip': true });
      `}</Script>
```

We should not use the same `id` and we should not use the same function name.